### ISSUE TITLE
Getter listening at array index not recomputing

### ISSUE THREAD

**Issue by m-mujica** [canjs/can-observable-array#59]
```js
class Foo extends StacheElement {
  static props = {
    profiles: {
      get default() {
        return new ObservableArray([ ... ]);
      }
    },
    get currentProfile() {
      return this.profiles[0] || this.emptyProfile;
    },

    get nextProfile() {
      return this.profiles[1] || this.emptyProfile;
    }
  };
}
customElements.define("a-foo", Foo);
```

both `currentProfile` and `nextProfile` compute fine on the first run but if I mutate `profiles` manually, like:

```js
document.querySelect('a-foo').profiles.unshift();
```

the getter won't ever return the `emptyProfile` property even when the array is empty.

**Comment by justinbmeyer** [comment]
Do we need to fix this?  `can-define` is able to work alright without it.  

I'm wary of a loop of 100 items creating 100 bindings.

**Comment by m-mujica** [comment]
Closed by #60
